core/import_data/import_replenishments.py

```diff
@@ -0,0 +1,127 @@
+import decimal
+import logging
+from decimal import Decimal
+
+import pandas as pd
+
+from core.models import Replenishment, Country, Contribution
+from core.import_data.utils import (
+    IMPORT_RESOURCES_DIR,
+    delete_old_data,
+)
+
+logger = logging.getLogger(__name__)
+
+COUNTRY_MAPPING = {
+    "Czech Republic": "Czechia",
+    "Slovak Republic": "Slovakia",
+    "Netherlands": "Netherlands (Kingdom of the)",
+    "United Kingdom": "United Kingdom of Great Britain and Northern Ireland",
+}
+
+REPLENISHMENT_YEARS = [(2021, 2023), (2024, 2026)]
+COUNTRY_COLUMN = 0
+UN_SCALE_OF_ASSESSMENT_COLUMN = 1
+ADJUSTED_SCALE_OF_ASSESSMENT_COLUMN = 2
+CONTRIBUTION_COLUMN = 3
+AVERAGE_INFLATION_RATE_COLUMN = 4
+QUALIFIES_FOR_FIXED_RATE_MECHANISM_COLUMN = 5
+FIXED_EXCHANGE_RATE_MECHANISM_COLUMN = 6
+FIXED_EXCHANGE_RATE_CURRENCY_COLUMN = 7
+FIXED_EXCHANGE_RATE_NATIONAL_CURRENCY_COLUMN = 8
+
+
+def decimal_converter(value):
+    try:
+        return Decimal(str(value))
+    except decimal.InvalidOperation:
+        return None
+
+
+def boolean_converter(value):
+    if pd.isna(value) or value == "N/A" or value == "Not Available" or pd.isnull(value):
+        return None
+    if value in {"0", 0}:
+        return False
+    return bool(value)
+
+
+def string_converter(value):
+    if (
+        pd.isna(value)
+        or value.strip() == "N/A"
+        or value.strip() == "Not Available"
+        or pd.isnull(value)
+    ):
+        return "N/A"
+    return str(value)
+
+
+def import_replenishments():
```

Comment:
I think is better if you add a [transaction.atomi](https://www.geeksforgeeks.org/transaction-atomic-with-django/)c decorator to this method

Reply:
Thanks! I totally forgot about that.